spec/spec.md

```diff
@@ -29,7 +29,7 @@ Participate:
 
 ## Abstract
 
-The JWT VC Presentation Profile defines a set of requirements against existing specifications to enable the interoperable presentation of [[ref:Verifiable Credentials]] (VCs) between [[ref: Wallets]] and [[ref: Verifiers]].
+The JWT VC Presentation Profile defines a set of requirements against existing specifications to enable the interoperable presentation of [[ref:Verifiable Credentials]] (VCs) between ref: Wallets and ref: Verifiers.
 
```

Comment:
remove these changes? since you update them in another PR and the ref: got weird here.

Reply:
i'll take care of this when merging all PRs